src/audio/kpb.c

```diff
@@ -740,8 +743,8 @@ static int kpb_buffer_data(struct comp_dev *dev,
 		current_time = platform_timer_get(timer);
 		if (timeout < current_time) {
 			trace_kpb_error("kpb_buffer_data(): timeout of %d [ms] (current state %d, state log %x)",
-					current_time - timeout, kpb->state,
-					kpb->state_log);
+							 current_time - timeout, kpb->state,
+							 kpb->state_log);
 			return -ETIME;
 		}
```

Comment:
Same here. This can easily be a separate patch which can be quickly merged as it is a cosmetic change.